web/views.py

```diff
@@ -11,6 +12,10 @@ def projects(request):
 	project_list = Project.objects.all()
 	return render(request, 'pages/projects.html',{'project_list': project_list})
 
+def tasks(request):
+	task_list = Task.objects.all()
```

Comment:
So even the tasks that are closed already and we don't need them grouped by project or ordered?